submissions/serializers.py

```diff
@@ -57,6 +57,29 @@ class TeamSubmissionSerializer(serializers.ModelSerializer):
     submitted_at = DateTimeField(format=None, required=False)
     created_at = DateTimeField(source='created', format=None, required=False)
     attempt_number = IntegerField(min_value=0)
+    # Prevent Django Rest Framework from converting the answer (dict or str)
+    # to a string.
```

Comment:
Maybe toss a comment on here saying that although this is included in the serializer that we save to a model, this value isn't a model field